server/base/src/main/java/org/apache/accumulo/server/util/CheckForMetadataProblems.java

```diff
@@ -90,19 +92,20 @@ private static void checkTable(TableId tableId, TreeSet<KeyExtent> tablets) {
       lastEndRow = tabke.endRow();
     }
     if (everythingLooksGood)
-      System.out.println("All is well for table " + tableId);
+      System.out.println("...All is well for table " + tableName + " (" + tableId + ")");
```

Comment:
If these "All is well" messages are indented with `...`, then the error messages should also be.